bitnami/fluentd/templates/servicemonitor.yaml

```
@@ -10,6 +10,13 @@ metadata:
     {{- range $key, $value := .Values.metrics.serviceMonitor.selector }}
     {{ $key }}: {{ $value | quote }}
     {{- end }}
+{{- if .Values.metrics.serviceMonitor.labels }}
+{{- toYaml .Values.metrics.serviceMonitor.labels | nindent 4 }}
+{{- end }}
+  {{- if .Values.metrics.serviceMonitor.annotations }}
+  annotations:
+{{- toYaml .Values.metrics.serviceMonitor.annotations | nindent 4 }}
+  {{- end }}
```

These lines seem to be wrongly indented, could you re-check them?
```suggestion
    {{- if .Values.metrics.serviceMonitor.labels }}
    {{- toYaml .Values.metrics.serviceMonitor.labels | nindent 4 }}
    {{- end }}
  {{- if .Values.metrics.serviceMonitor.annotations }}
  annotations: {{- toYaml .Values.metrics.serviceMonitor.annotations | nindent 4 }}
  {{- end }}
```

---

Well, it looks good to me. The linter is OK

---

The problem is that they are not indented consistenly (i.e. check the rest of the file). Could you change it? Thanks!

---

Sorry, I was missing the definition of `nindent` over `indent`.
Thanks for the head-up!
It's fixed now.